latch_cli/snakemake/single_task_snakemake.py

```diff
@@ -28,6 +29,11 @@ def eprint(x: str) -> None:
     print(x, file=sys.stderr)
 
 
+print_compilation_env = os.environ["LATCH_PRINT_COMPILATION"]
+print_compilation = False
+if print_compilation_env == "1":
+    print_compilation = True
```

Comment:
```suggestion
print_compilation = False
if os.environ.get("LATCH_PRINT_COMPILATION") == "1":
    print_compilation = True
```

Reply:
actually why get this from the env in the first place - seems like an unnecessary switch considering that the env variable is always being set anyway

just have it print-compilation always

Reply:
its not always being set, its only being set when we want to print the compilation